tests/openvino/utils_tests.py

```diff
@@ -118,6 +118,7 @@
     "stable-diffusion-openvino": "hf-internal-testing/tiny-stable-diffusion-openvino",
     "stable-diffusion-xl": "echarlaix/tiny-random-stable-diffusion-xl",
     "stable-diffusion-xl-refiner": "echarlaix/tiny-random-stable-diffusion-xl-refiner",
+    "stable-diffusion-3": "yujiepan/stable-diffusion-3-tiny-random",
```

Comment:
in the exporters and exporters cli as well:
https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_exporters_cli.py#L90-L94
https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_export.py#L67-L71

Reply:
added